src/software/sensor_fusion/sensor_fusion.cpp

```diff
@@ -87,10 +87,7 @@ void SensorFusion::updateWorld(const Referee &packet)
 {
     // TODO remove DynamicParameters as part of
     // https://github.com/UBC-Thunderbots/Software/issues/960
-    if (DynamicParameters->getAIControlConfig()
-            ->getRefboxConfig()
-            ->FriendlyColorYellow()
-            ->value())
+    if (DynamicParameters->getSensorFusionConfig()->FriendlyColorYellow()->value())
```

Comment:
Why not make SensorFusionConfig a field in SensorFusion?

Reply:
Originally was just moving params and not focusing on injecting them. Made it a field and replaced usages with the member variable.